hasjob/views/listing.py

```diff
@@ -162,13 +160,31 @@ def jobdetail(domain, hashid):
 
     is_bgroup = getbool(request.args.get('b'))
     headline = post.headlineb if is_bgroup and post.headlineb else post.headline
+    is_post_admin = is_siteadmin or post.admin_is(g.user) or (g.user and g.user.flags.is_employer_month)
```

Comment:
`JobPost` already has `admin_is()`. I think checking `g.user.is_employer_month` inside `JobPost` will be stretching it a little. It's better off here.